Adyen.Test/CheckoutTest.cs

```diff
@@ -674,14 +674,14 @@ public void DragonpayDetailsDeserializationTest()
             Assert.AreEqual(result.Type, DragonpayDetails.TypeEnum.Ebanking);
         }
 
-        [TestMethod]
+        /*[TestMethod]
```

Comment:
commented code

Reply:
This payment method was removed in checkout, I'll delete the test instead of the comment